javascript/index.js

```diff
@@ -1,22 +1,306 @@
+                <li class="css">${details[2].tech3}</li>
+                <li class="js">${details[2].tech2}</li>
+                <li class="html">${details[2].tech1}</li>
+            </ul>
+            <div class="btn-container">
+                <button type="button" class="btn" data-card = 'card2'>See Project</button>
+            </div>
+        </div>
+        <div class="sample-containers">
+            <div class="sample-img-container">
+                <img src=${details[3].imgSrc} alt="" class="sample-img" id="sample-img-1">
+            </div>
+            <div class="sample-img-txt-head-container">
+                <h3 class="sample-img-head-txt">${details[3].projectName}</h3>
+            </div>
+            <ul class="code-lang-container">
+                <li class="ruby-on-rails">${details[3].tech4}</li>
+                <li class="css">${details[3].tech3}</li>
+                <li class="js">${details[3].tech2}</li>
+                <li class="html">${details[3].tech1}</li>
+            </ul>
+            <div class="btn-container">
+                <button type="button" class="btn" data-card= 'card3'>See Project</button>
+            </div>
+        </div>   
+        <div class="sample-containers">
+            <div class="sample-img-container">
+                <img src=${details[4].imgSrc} alt="" class="sample-img" id="sample-img-1">
+            </div>
+            <div class="sample-img-txt-head-container">
+                <h3 class="sample-img-head-txt">${details[4].projectName}</h3>
+            </div>
+            <ul class="code-lang-container">
+                <li class="ruby-on-rails">${details[4].tech4}</li>
+                <li class="css">${details[4].tech3}</li>
+                <li class="js">${details[4].tech2}</li>
+                <li class="html">${details[4].tech1}</li>
+            </ul>
+            <div class="btn-container">
+              <button type="button" class="btn" data-card= 'card4'>See Project</button>
+            </div>
+        </div>
+        <div class="sample-containers">
+            <div class="sample-img-container">
+                <img src=${details[5].imgSrc} alt="" class="sample-img" id="sample-img-1">
+            </div>
+            <div class="sample-img-txt-head-container">
+                <h3 class="sample-img-head-txt">${details[5].projectName}</h3>
+            </div>
+            <ul class="code-lang-container">
+                <li class="ruby-on-rails">${details[5].tech4}</li>
+                <li class="css">${details[5].tech3}</li>
+                <li class="js">${details[5].tech2}</li>
+                <li class="html">${details[5].tech1}</li>
+            </ul>
+            <div class="btn-container">
+                <button type="button" class="btn" data-card= 'card5'>See Project</button>
+            </div>
+        </div>           
+    </article>`;
+}
+populateMain();
+
+function openCloseProjDetails(event) {
```

Comment:
- [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```
- [ ] <b>REQUIRED:<b/> On desktop, other details of modal are overlapped because the height is too small. kindly fix that.
![scrnli_9_3_2021_12-31-58 AM](https://user-images.githubusercontent.com/45217364/131924901-9d785d36-d664-4901-99ad-b2152d4889e0.png)

Reply:
This is the same as the first. As for the height I had problems with making it fit multiple screens so I had to give it a height so I could give it an overflow. But I made it better now.